src/applications/vaos/appointment-list/sass/styles.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../../../../node_modules/@department-of-veterans-affairs/formation/sass/shared-variables";
```

Comment:
I think you only need these imports in the main file? Everything is still getting bundled into one css file.

Reply:
Only thing you loose is the ide hinting I believe.

Reply:
Fixed.